index.js

```diff
@@ -72,8 +67,8 @@ if (electron && nvm && !fs.existsSync(globals)) {
  * @param {LoadPluginOptions} [options]
  * @returns {any}
  */
-function loadPlugin(name, options) {
-  return require(resolvePlugin(name, options) || name)
+export default function loadPlugin(name, options) {
```

Comment:
Tests rely on this being the default export. I personally prefer a named export, especially for packages that export multiple members.